packages/framer-motion/src/value/index.ts

```diff
@@ -171,28 +157,34 @@ export class MotionValue<V = any> {
      * @param subscriber - A function that receives the latest value.
      * @returns A function that, when called, will cancel this subscription.
      *
-     * @public
+     * @deprecated
      */
     onChange(subscription: Subscriber<V>): () => void {
```

Comment:
I guess there's my answer 😁

Reply:
Ha. I think it's so widely used by us I won't even throw in a `warning` yet. I'm gonna drop it in the docs and then revisit in 6 months or so, maybe add a warning then.